src/index.js

```diff
@@ -9,7 +9,7 @@ import functionArgNames from './functionArgNames';
 
 let actionNum = 0;
 const getActionType = (prefix, actionName) =>
-(prefix != null ? prefix : "") +
+(prefix ? prefix + "/" : "") +
```

Comment:
Yes, might as well standardize the prefix along the duck convention. But maybe it'd be nice with a check for whether there is already a slash at the end? I'll pull this and make that. :)